keyboards/acheron/austin/info.json

```diff
@@ -0,0 +1,83 @@
+                {"label":"-", "x":15, "y":0},
+                {"label":"=", "x":16, "y":0.25},
+                {"label":"Del", "x":17, "y":0.25},
+                {"label":"Backspace", "x":18, "y":0.25},
+
+                {"label":"PgUp", "x":0.25, "y":1},
+                {"label":"Tab", "x":1.5, "y":1.25, "w":1.5},
+                {"label":"Q", "x":3, "y":1.25},
+                {"label":"W", "x":4.25, "y":1.25},
+                {"label":"E", "x":5.25, "y":1.25},
+                {"label":"R", "x":6.25, "y":1.25},
+                {"label":"T", "x":7.25, "y":1.25},
+                {"label":"Y", "x":10.5, "y":1.25},
+                {"label":"U", "x":11.5, "y":1.25},
+                {"label":"I", "x":12.5, "y":1.25},
+                {"label":"O", "x":13.5, "y":1.25},
+                {"label":"P", "x":14.75, "y":1.25},
+                {"label":"{", "x":15.75, "y":1.25},
+                {"label":"}", "x":16.75, "y":1.25},
+                {"label":"|", "x":17.75, "y":1.25, "w":1.5},
+
+                {"label":"PgDn", "x":0, "y":2},
+                {"label":"Caps Lock", "x":1.5, "y":2.25, "w":1.75},
+                {"label":"A", "x":3.25, "y":2.25},
+                {"label":"S", "x":4.5, "y":2.25},
+                {"label":"D", "x":5.5, "y":2.25},
+                {"label":"F", "x":6.5, "y":2.25},
+                {"label":"G", "x":7.5, "y":2.25},
+                {"label":"H", "x":10.75, "y":2.25},
+                {"label":"J", "x":11.75, "y":2.25},
+                {"label":"K", "x":12.75, "y":2.25},
+                {"label":"L", "x":13.75, "y":2.25},
+                {"label":":", "x":15.25, "y":2.25},
+                {"label":"\"", "x":16.25, "y":2.25},
+                {"label":"Enter", "x":17.25, "y":2.25, "w":2.25},
+
+                {"label":"Shift", "x":1.5, "y":3.25, "w":2.25},
+                {"label":"Z", "x":3.75, "y":3.25},
+                {"label":"X", "x":5, "y":3.25},
+                {"label":"C", "x":6, "y":3.25},
+                {"label":"V", "x":7, "y":3.25},
+                {"label":"B", "x":8, "y":3.25},
+                {"label":"Fn", "x":10.25, "y":3.25},
+                {"label":"N", "x":11.25, "y":3.25},
+                {"label":"M", "x":12.25, "y":3.25},
+                {"label":"<", "x":13.25, "y":3.25},
+                {"label":">", "x":15, "y":3.25},
+                {"label":"?", "x":16, "y":3.25},
+                {"label":"Shift", "x":17, "y":3.25, "w":1.75},
+                {"label":"Fn", "x":18.75, "y":3.25},
+
+                {"label":"Ctrl", "x":1.5, "y":4.25, "w":1.5},
+                {"label":"Alt", "x":5, "y":4.25, "w":1.5},
+                {"label":"Space", "x":6.5, "y":4.25, "w":2},
+                {"label":"Menu", "x":8.5, "y":4.25},
+                {"label":"Space", "x":10.25, "y":4.25, "w":2.25},
+                {"label":"Alt", "x":12.5, "y":4.25, "w":1.5},
+                {"label":"Ctrl", "x":18, "y":4.25, "w":1.5}
+            ]
+        }
```

Comment:
## Changset A3

Please replace Lines 8-81 with the following:

```json
        "LAYOUT_all": {
            "key_count": 105,
            "layout": [
                {"label":"Esc", "x":0, "y":0},
                {"label":"F1", "x":1.25, "y":0},
                {"label":"F2", "x":2.25, "y":0},
                {"label":"F3", "x":3.25, "y":0},
                {"label":"F4", "x":4.25, "y":0},
                {"label":"F5", "x":5.5, "y":0},
                {"label":"F6", "x":6.5, "y":0},
                {"label":"F7", "x":7.5, "y":0},
                {"label":"F8", "x":8.5, "y":0},
                {"label":"F9", "x":9.75, "y":0},
                {"label":"F10", "x":10.75, "y":0},
                {"label":"F11", "x":11.75, "y":0},
                {"label":"F12", "x":12.75, "y":0},
                {"label":"Print Screen", "x":14, "y":0},
                {"label":"Scroll Lock", "x":15.25, "y":0},
                {"label":"Pause", "x":16.5, "y":0},
                {"label":"Home", "x":17.5, "y":0},
                {"label":"End", "x":18.5, "y":0},
                {"label":"Delete", "x":19.5, "y":0},

                {"label":"`", "x":0, "y":1.25},
                {"label":"1", "x":1, "y":1.25},
                {"label":"2", "x":2, "y":1.25},
                {"label":"3", "x":3, "y":1.25},
                {"label":"4", "x":4, "y":1.25},
                {"label":"5", "x":5, "y":1.25},
                {"label":"6", "x":6, "y":1.25},
                {"label":"7", "x":7, "y":1.25},
                {"label":"8", "x":8, "y":1.25},
                {"label":"9", "x":9, "y":1.25},
                {"label":"0", "x":10, "y":1.25},
                {"label":"-", "x":11, "y":1.25},
                {"label":"=", "x":12, "y":1.25},
                {"label":"Backspace", "x":13, "y":1.25},
                {"label":"Backspace", "x":14, "y":1.25},
                {"label":"Insert", "x":15.25, "y":1.25},
                {"label":"Num Lock", "x":16.5, "y":1.25},
                {"label":"/", "x":17.5, "y":1.25},
                {"label":"*", "x":18.5, "y":1.25},
                {"label":"-", "x":19.5, "y":1.25},

                {"label":"Tab", "x":0, "y":2.25, "w":1.5},
                {"label":"Q", "x":1.5, "y":2.25},
                {"label":"W", "x":2.5, "y":2.25},
                {"label":"E", "x":3.5, "y":2.25},
                {"label":"R", "x":4.5, "y":2.25},
                {"label":"T", "x":5.5, "y":2.25},
                {"label":"Y", "x":6.5, "y":2.25},
                {"label":"U", "x":7.5, "y":2.25},
                {"label":"I", "x":8.5, "y":2.25},
                {"label":"O", "x":9.5, "y":2.25},
                {"label":"P", "x":10.5, "y":2.25},
                {"label":"[", "x":11.5, "y":2.25},
                {"label":"]", "x":12.5, "y":2.25},
                {"label":"\\", "x":13.5, "y":2.25, "w":1.5},
                {"label":"Page Up", "x":15.25, "y":2.25},
                {"label":"7", "x":16.5, "y":2.25},
                {"label":"8", "x":17.5, "y":2.25},
                {"label":"9", "x":18.5, "y":2.25},
                {"label":"+", "x":19.5, "y":2.25, "h":2},

                {"label":"Caps Lock", "x":0, "y":3.25, "w":1.75},
                {"label":"A", "x":1.75, "y":3.25},
                {"label":"S", "x":2.75, "y":3.25},
                {"label":"D", "x":3.75, "y":3.25},
                {"label":"F", "x":4.75, "y":3.25},
                {"label":"G", "x":5.75, "y":3.25},
                {"label":"H", "x":6.75, "y":3.25},
                {"label":"J", "x":7.75, "y":3.25},
                {"label":"K", "x":8.75, "y":3.25},
                {"label":"L", "x":9.75, "y":3.25},
                {"label":";", "x":10.75, "y":3.25},
                {"label":"'", "x":11.75, "y":3.25},
                {"label":"Enter", "x":12.75, "y":3.25, "w":2.25},
                {"label":"Page Down", "x":15.25, "y":3.25},
                {"label":"4", "x":16.5, "y":3.25},
                {"label":"5", "x":17.5, "y":3.25},
                {"label":"6", "x":18.5, "y":3.25},

                {"label":"Shift", "x":0, "y":4.25, "w":1.25},
                {"label":"ISO \\", "x":1.25, "y":4.25},
                {"label":"Z", "x":2.25, "y":4.25},
                {"label":"X", "x":3.25, "y":4.25},
                {"label":"C", "x":4.25, "y":4.25},
                {"label":"V", "x":5.25, "y":4.25},
                {"label":"B", "x":6.25, "y":4.25},
                {"label":"N", "x":7.25, "y":4.25},
                {"label":"M", "x":8.25, "y":4.25},
                {"label":",", "x":9.25, "y":4.25},
                {"label":".", "x":10.25, "y":4.25},
                {"label":"/", "x":11.25, "y":4.25},
                {"label":"Shift", "x":12.25, "y":4.25, "w":1.75},
                {"label":"Up", "x":14.25, "y":4.5},
                {"label":"1", "x":16.5, "y":4.25},
                {"label":"2", "x":17.5, "y":4.25},
                {"label":"3", "x":18.5, "y":4.25},
                {"label":"Enter", "x":19.5, "y":4.25, "h":2},

                {"label":"Ctrl", "x":0, "y":5.25, "w":1.25},
                {"label":"GUI", "x":1.25, "y":5.25, "w":1.25},
                {"label":"Alt", "x":2.5, "y":5.25, "w":1.25},
                {"label":"Space", "x":3.75, "y":5.25, "w":6.25},
                {"label":"Alt", "x":10, "y":5.25, "w":1.5},
                {"label":"Ctrl", "x":11.5, "y":5.25, "w":1.5},
                {"label":"Left", "x":13.25, "y":5.5},
                {"label":"Down", "x":14.25, "y":5.5},
                {"label":"Right", "x":15.25, "y":5.5},
                {"label":"0", "x":16.5, "y":5.25},
                {"label":"0", "x":17.5, "y":5.25},
                {"label":".", "x":18.5, "y":5.25}
            ]
        }
```

Updates your QMK Configurator implementation per A1.